libs/core/langchain_core/messages/ai.py

```diff
@@ -51,6 +51,10 @@ class UsageMetadata(TypedDict):
     """Count of output (or completion) tokens."""
     total_tokens: int
     """Total token count."""
+    cache_creation_input_tokens: NotRequired[int]
```

Comment:
i dont think we want to add this to core until at least one or two other providers support a similar feature